templates/js-cypress/main.js

```diff
@@ -0,0 +1,49 @@
+import { Actor } from 'apify';
+import fs from 'fs';
+import cypress from 'cypress';
+import { globby } from 'globby';
+import log from '@apify/log';
+
+await Actor.init();
+
+const input = await Actor.getInput();
+
+const runOneSpec = (spec) => {
```

Comment:
I'd prefer:

`const runOneSpec = (spec) => cypress.run({ config: input, spec });`

but up to you